weaviate/graphql/nearobjectbuilder.go

```diff
@@ -82,3 +86,27 @@ func (e *NearObjectArgumentBuilder) build() string {
 	}
 	return fmt.Sprintf("nearObject:{%s}", strings.Join(clause, " "))
 }
+
+func (e *NearObjectArgumentBuilder) togrpc() *pb.NearObject {
```

Comment:
```suggestion
func (b *NearObjectArgumentBuilder) togrpc() *pb.NearObject {
```

suggestion: rename receiver to `b` for Builder

Reply:
in this file we use `e`, I will adjust all other builder methods to `b`

Reply:
done